src/test/create-user.test.ts

```diff
@@ -0,0 +1,113 @@
+import axios from "axios";
+import { expect } from "chai";
+import { dataSource } from "../data-source";
+import { User } from "../entity/User";
+import * as bcrypt from "bcrypt";
+
+describe("Mutation createUser", () => {
+  const mutation = `mutation CreateUser($data: UserInput!) {
+        createUser(data: $data) {
+          id
+          email
+          name
+          birthDate
+        }
+      }
+      `;
+  const urlDB = "http://localhost:4000/graphql";
+  const USER_INPUT = {
```

Comment:
o input pode deixar como let e preencher dentro de cada it, pois vc pode querer mudar a entrada para cada teste. E pode deixar o nome de input msm sem snakes/caps haha